dominion/dominion_ui/src/components/Header/index.js

```diff
@@ -0,0 +1,174 @@
+import React, { Component } from 'react';
```

Comment:
Right align menu open/close button (should be inline with search button below it).

![screenshot from 2019-02-27 13-13-36](https://user-images.githubusercontent.com/1779590/53483124-d7ff1c00-3a91-11e9-9d11-b12267fd9fd0.png)

Make mobile view start from `sm` i.e. anything below [960px wide](https://material-ui.com/layout/breakpoints/). This header (and most of the components) won't fit in a 600-959px screen.

![screenshot from 2019-02-27 13-20-28](https://user-images.githubusercontent.com/1779590/53483421-84410280-3a92-11e9-8032-b9c916d659ba.png)